tests/locales.test.ts

```diff
@@ -2,6 +2,7 @@ import { Brazil_PtBr } from 'romcal/dist/bundles/brazil';
 import { France_Fr } from 'romcal/dist/bundles/france';
 
 import { Romcal } from '../lib';
+import { fixture as nameFixtures } from './fixtures/names.fixture';
```

Comment:
this makes sense to me. I find that I make heavy use of aliases so as to use the same name out of a lot of exports so that the consumer (in this case, the test) can alias as needed and we don't have to make up unnecessarily complex unique names in each file/implementation/etc, especially where the name of the file generally gives this differentiation

Reply:
```suggestion
import { namesFixture } from './fixtures/names.fixture';
```

Reply:
> we don't have to make up unnecessarily complex unique names in each file/implementation/etc, especially where the name of the file generally gives this differentiation

While I get what you mean, however, I prefer the filename to be based on the export name in kebab-case (without the suffix when it is part of the export name) with a suffix and extension (e.g. `namesFixture` → `names.fixture.ts`).

That said, I am open to define any export naming style, even one that I dislike (e.g. I don’t use semicolons unless necessary, however, Etienne uses them, thus they are used in romcal).

Reply:
oh, i meant more that, like you, i have tendencies wherein i'm willing to do something different from them